python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -11,12 +11,10 @@
 from cudf.core._compat import PANDAS_GE_120
 
 import dask_cudf
-from dask_cudf.groupby import SUPPORTED_AGGS, _aggs_supported
+from dask_cudf.groupby import AGGS, _aggs_supported, CUMULATIVE_AGGS
```

Comment:
Following up on the last comment:

```suggestion
from dask_cudf.groupby import AGGS, CUMULATIVE_AGGS, SUPPORTED_AGGS, _aggs_supported
```